### PR TITLE
ignore empty lines in ics parse

### PR DESCRIPTION
**Pull request by ericdude4**
https://github.com/lpil/icalendar/issues/40

### CODE REVIEW

File: test/icalendar/util/deserialize_test.exs [hunk 1]
```diff
@@ -128,4 +128,12 @@ defmodule ICalendar.Util.DeserializeTest do
...
            ]
   end
+
+  test "ignore empty lines" do
+    line = ""
+
+    kvs = Deserialize.retrieve_kvs(line)
+
+    assert kvs == nil
+  end
```
Comment (lpil) [inline test/icalendar/util/deserialize_test.exs:161]: Can we have a test that shows why there would be empty lines? i.e. a full ics string being parsed

File: lib/icalendar/util/deserialize.ex [hunk 2]
```diff
@@ -9,7 +9,13 @@ defmodule ICalendar.Util.Deserialize do
   def build_event(lines) when is_list(lines) do
     lines
```
Comment (lpil) [inline lib/icalendar/util/deserialize.ex:10]: Could we instead filter empty lines here?

```elixir
|> Enum.filter(& &1 == "")
```

File: test/icalendar/util/deserialize_test.exs (hunk 1 above)
Comment (ericdude4) [reply test/icalendar/util/deserialize_test.exs:161]: I included a sample from the VEVENT that was breaking my application.

**Review by lpil — APPROVED** [review]
Thank you

**Pull request merged**